core/src/main/java/hudson/model/ComputerSet.java

```diff
@@ -106,15 +113,36 @@ public static List<NodeMonitor> get_monitors() {
         return monitors.toList();
     }
 
-    @Exported(name = "computer", inline = true)
+    /**
+     * @deprecated Use {@link #getComputers()} instead.
+     * @return All {@link Computer} instances managed by this set.
+     */
+    @Deprecated(since = "TODO")
     public Computer[] get_all() {
-        return Jenkins.get().getComputers();
+        return getComputers().stream().filter(Computer.class::isInstance).toArray(Computer[]::new);
+    }
+
+    /**
+     * @return All {@link IComputer} instances managed by this set.
+     */
+    @Exported(name = "computer", inline = true)
+    public Collection<? extends IComputer> getComputers() {
```

Comment:
There might be a small change in the API with this, as `Collection`  does not grant ordering while the previous signature with an array was actually doing it. I don't think this will affect any client of this API, but just noting.

Reply:
array doesn't grant it either, however indeed the sort call is missing from the new implementation. Will fix it.